lib/shoulda/matchers/integrations/test_frameworks/rspec_expectations.rb

```diff
@@ -0,0 +1,34 @@
+module Shoulda
+  module Matchers
+    module Integrations
+      module TestFrameworks
+        # @private
+        class RspecExpectations
+          Integrations.register_test_framework(self, :rspec_exp)
+
+          def validate!
+            return if defined? RSpec::Matchers
+            raise TestFrameworkNotFound, <<-EOT
+You need to include the 'rspec-expectations' gem to Gemfile, and add
+the line before requiring the shoulda matchers:
+
+require 'rspec/expectations'
+EOT
+          end
+
+          def include(*modules, **_options)
+            RSpec::Matchers.send(:include, *modules)
```

Comment:
So now, instead of including modules into RSpec::Matchers (which requires understanding that RSpec::Matchers is itself mixed into Object), this could be `World(*modules)` (which is a public API).